test/NosCore.Tests/GuriHandlerTests/SpeakerGuriHandlerTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿//  __  _  __    __   ___ __  ___ ___
+// |  \| |/__\ /' _/ / _//__\| _ \ __|
+// | | ' | \/ |`._`.| \_| \/ | v / _|
+// |_|\__|\__/ |___/ \__/\__/|_|_\___|
+// 
+// Copyright (C) 2019 - NosCore
+// 
+// NosCore is a free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using ChickenAPI.Packets.ClientPackets.Inventory;
+using ChickenAPI.Packets.Enumerations;
+using ChickenAPI.Packets.ServerPackets.Chats;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using NosCore.Data;
+using NosCore.Data.Enumerations.Items;
+using NosCore.Data.StaticEntities;
+using NosCore.GameObject;
+using NosCore.GameObject.Networking;
+using NosCore.GameObject.Providers.GuriProvider.Handlers;
+using NosCore.GameObject.Providers.InventoryService;
+using NosCore.GameObject.Providers.ItemProvider;
+using NosCore.GameObject.Providers.ItemProvider.Item;
+using NosCore.Tests.Helpers;
+using Serilog;
+using GuriPacket = ChickenAPI.Packets.ClientPackets.UI.GuriPacket;
+
+namespace NosCore.Tests.GuriHandlerTests
+{
+    [TestClass]
+    public class SpeakerGuriHandlerTests : GuriEventHandlerTestsBase
+    {
+        private IItemProvider _itemProvider;
+        private Mock<ILogger> _logger;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_logger' and declare it as a local variable in the relevant methods.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4628412)